src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function Person(props) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props;
+
+  return (
+    <section className="Person">
+      <h1 className="Person__name">
+        {`My name is ${name}`}
+      </h1>
+
+      <p className="Person__age">
+        {age ? `I am ${age}` : null}
```

Comment:
1. add conditional rendering for the whole p tag
2. use logical operator, not ternary